### PR TITLE
modify walkthrough

### PR DESCRIPTION
**Pull request by jcolomb**
added more detailed walkthrough, making it easier to know what is done on google and what is done in the app

### CODE REVIEW

File: inst/app/www/introduction.Rmd [hunk 1]
```diff
@@ -4,13 +4,15 @@ output: html_document
 ---
 
 #### 1. Create your contributors table
-* Copy the contributors table template in your Google Drive _File -> Make a copy_
+* Follow the link in the app, copy the contributors table googlesheet template in your Google Drive _File -> Make a copy_
```
Comment (marton-balazs-kovacs) [inline inst/app/www/introduction.Rmd:7]: This page is already on the how to use tenzing tab of the app. So I am not sure that I understand the follow the link in that app part.

Comment (jcolomb) [reply inst/app/www/introduction.Rmd:7]:
```suggestion
* Follow the link in the first tab of the app, copy the contributors table googlesheet template in your Google Drive _File -> Make a copy_
```

Comment (jcolomb) [reply inst/app/www/introduction.Rmd:7]: that would be better ?

Comment (marton-balazs-kovacs) [inline inst/app/www/introduction.Rmd:7]:
```suggestion
* Copy the [contributors table googlesheet template](https://docs.google.com/spreadsheets/d/1Gl0cwqN_nTsdFH9yhSvi9NypBfDCEhViGq4A3MnBrG8/edit?usp=sharing) in your Google Drive _File -> Make a copy_
```

What about this?

**Comment by alexholcombe** [comment]
This is now outdated due to other changes.

**Pull request closed without merging**